### PR TITLE
Command Palette

### LINKED ISSUE #728: Introduce keyboard navigation via ctr+k (⌘ k)
As [Sebastian](https://github.com/kyma-project/community/blob/main/manifesto/personas.md#sebastian---tech-lead) and [Mathew](https://github.com/kyma-project/community/blob/main/manifesto/personas.md#matthew---cloud-native-software-engineer) I would like to be able to use the keyboard to navigate busola menu similar to Github using ⌘ k

image

**Acceptance Criteria**

- [x] Follow Project DOD
- [x] Introduce a ⌘ ? that will show the user the available commands
- [x] the commands will follow the `kubectl` short commands ([shortNames](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#create-a-customresourcedefinition))
- [x] This task covers basic command lines from [the SHORTNAMES list](https://kubernetes.io/docs/reference/kubectl/overview/#resource-types) and only navigates to list view
- [x] testing is performed to smoke test the feature
- [x] Absolutely great to have https://www.npmjs.com/package/didyoumean

**Examples**
- [react-command-palette](https://github.com/asabaylus/react-command-palette)
- [gitkraken](https://support.gitkraken.com/start-here/command-palette/)
- [react-super-cmd](https://saharmor.github.io/react-super-cmd/)
- [kbar](https://github.com/timc1/kbar)
- ninja-keys](https://github.com/ssleptsov/ninja-keys)
- [scoutbar](https://github.com/adenekan41/scoutbar)
**Tasks**

- [ ] Add Tasks

### PR DESCRIPTION
**Pull request by Wawrzyn321**
**Description**

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**

Resolves #728

### CODE REVIEW

File: core-ui/src/Compass/CompassUI/handlers/logsHandler.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+        LuigiClient.linkManager()
+          .fromContext('cluster')
+          .navigate(
+            `${namespacePart}/pods/details/${podName}/containers/${containerName}`,
+          ),
+    };
+  });
+}
+
+// todo wtf naming
```
[the remark this answers is not in the document]
Comment (Wawrzyn321) [reply core-ui/src/Compass/CompassUI/handlers/logsHandler.js:71]: oh no

File: core-ui/src/Compass/CompassUI/handlers/crdHandler.js [hunk 2]
```diff
@@ -55,12 +55,12 @@ function makeListItem(item, namespace, t) {
   };
 }
 
-function isAboutCRDs({ tokens }) {
+function concertnsCRDs({ tokens }) {
```
Comment (qbalukom) [inline core-ui/src/Compass/CompassUI/handlers/crdHandler.js:58]: shouldn't this be "concernsCRDs"?

**Review by qbalukom — DISMISSED** [review]

**Review by dariadomagala — APPROVED** [review]

**Pull request merged**